tests/data/hostgaplugin/ext_conf.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm a bit confused by the inclusion of this file-- it doesn't quite fit the PR name. Is it supposed to be here?

Reply:
it's used by the unit test